credsweeper/__main__.py

```diff
@@ -107,14 +107,16 @@ def get_json_filenames(json_filename: str):
 
 
 def scan(args, content_provider, json_filename):
-    credsweeper = CredSweeper(rule_path=args.rule_path,
-                              ml_validation=args.ml_validation,
-                              api_validation=args.api_validation,
-                              json_filename=json_filename,
-                              pool_count=args.jobs,
-                              ml_batch_size=args.ml_batch_size,
-                              ml_threshold=args.ml_threshold,
-                              find_by_ext=args.find_by_ext)
+    credsweeper = CredSweeper(
+        rule_path=args.rule_path,
+        ml_validation=args.ml_validation,
+        api_validation=args.api_validation,
+        json_filename=json_filename,
+        pool_count=args.jobs,
+        ml_batch_size=args.ml_batch_size,
+        ml_threshold=args.ml_threshold,
+        find_by_ext=args.find_by_ext,
+    )
```

Comment:
Please check this formatting 
It seems this conflicting our yapf config

https://github.com/google/yapf see `INDENT_CLOSING_BRACKETS` `DEDENT_CLOSING_BRACKETS`
dedent is false in our config https://github.com/Samsung/CredSweeper/blob/b960eb3304caa1652bd9e8922e1f1487e3dbb8ad/.style.yapf#L116